.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,28 +10,17 @@ labels: Bug
 
 ### Steps to Reproduce
 
+<!-- Describe the specific steps on how to reproduce the issue. -->
+
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-### Expected Behavior
-
-<!-- A clear and concise description of what you expected to happen. -->
+### Related Issues
 
-### Actual Behavior
-
-<!-- A clear and concise description of what actually happened. -->
+<!-- Link all related issues which are e.g. CMS issues or otherwise related, blocking, duplicating issues, ... -->
 
 ### Additional Information
 
 <!-- Add any other context (e.g. logs, screenshots, environment, related issues etc.) about the problem here. -->
-
-<details>
-  <summary>Traceback</summary>
-
-```
-
-```
-
-</details>
```

Comment:
I think I have never seen a single issue using this but I think it was really annoying to fill this.